shims/common/src/main/scala/org/apache/gluten/expression/ExpressionNames.scala

```diff
@@ -246,6 +246,8 @@ object ExpressionNames {
   final val ARRAY_REPEAT = "array_repeat"
   final val ARRAY_REMOVE = "array_remove"
   final val FILTER = "filter"
+  final val ALL_MATCH = "all_match"
+  final val ANY_MATCH = "any_match"
```

Comment:
shims module is common for all backends, we should use vanilla spark's function name here, and do conversion in each backends internal if needed.

so, I expect here should be `forall` and `exist`.

Reply:
Fixed